src/sentry/api/endpoints/organization_index.py

```diff
@@ -220,23 +216,24 @@ def post(self, request: Request) -> Response:
             result = serializer.validated_data
 
             try:
+                create_default_team = bool(result.get("defaultTeam"))
+                provision_args = OrganizationProvisioningOptions(
+                    provision_options=OrganizationOptions(
+                        name=result["name"],
+                        slug=result.get("slug") or result["name"],
+                        owning_user_id=request.user.id,
+                        create_default_team=create_default_team,
+                    ),
+                    post_provision_options=PostProvisionOptions(
+                        getsentry_options=None, sentry_options=None
+                    ),
+                )
 
-                with transaction.atomic(router.db_for_write(Organization)):
-                    org = create_organization_with_outbox_message(
-                        create_options={"name": result["name"], "slug": result.get("slug")}
-                    )
-                    om = OrganizationMember.objects.create(
-                        organization_id=org.id,
-                        user_id=request.user.id,
-                        role=roles.get_top_dog().id,
-                    )
-
-                    if result.get("defaultTeam"):
-                        team = org.team_set.create(name=org.name)
-
-                        OrganizationMemberTeam.objects.create(
-                            team=team, organizationmember=om, is_active=True
-                        )
+                rpc_org = organization_provisioning_service.provision_organization(
+                    region_name=settings.SENTRY_MONOLITH_REGION,
```

Comment:
Nice, now we have a slot to fill in when we add the region selector input :clap: